Chromium/Makefile

```diff
@@ -1,9 +1,6 @@
 all: tables guide content dist
 
-# Do not remove exporting of SHARED variable!!! (since
-# shared/transforms/combineremediations.py helper
-# requires its value to be available)
-export SHARED = ../shared
```

Comment:
@mpreisler don't we get the $SHARED_RP param/variable for XSLT from this export?